.github/workflows/releases.yml

```diff
@@ -26,6 +26,22 @@ jobs:
           args: release --release-notes=CHANGELOG.md
         env:
           GITHUB_TOKEN: ${{secrets.UPLOAD_GITHUB_TOKEN}}
+  cards:
```

Comment:
Does this need to be a separate job? Since it also runs on ubuntu-latest and depends on goreleaser having completed, could we simply move it to a step in the existing? `goreleaser` job? Steps can also have an `if` condition.

Reply:
i'm fine with that! i wasn't sure how "correct" to be.